docs/deployment-process/execution-containers-for-workers/index.md

```diff
@@ -1,25 +1,55 @@
 ---
 title: Execution containers for workers
-description: How to set a step in your deployment process to run inside a container.
+description: How to set a step in your deployment/runbook process to run inside a container.
 position: 15
 ---
 
-For a [step](/docs/deployment-process/steps/index.md) running on a [worker](/docs/infrastructure/workers/index.md) or on [Octopus server](/docs/infrastructure/workers/built-in-worker.md) that you define in your [deployment processes](/docs/deployment-process/index.md), you can select a container for the step's execution. 
+For a [step](/docs/deployment-process/steps/index.md) running on a [worker](docs/infrastructure/workers/index.md) or on [Octopus server](docs/infrastructure/workers/built-in-worker.md) that you define in your [deployment processes](/docs/deployment-process/index.md), you can select a container for the step's execution.
 
-## Minimum requirements for your step to run in a container
-The worker will need Docker installed. 
+## Minimum requirements
+You will need Docker installed and running, on the worker in order to use execution containers for workers
 
-## Setup a container registry and an image 
 
-- Set a [feed](/docs/packaging-applications/package-repositories/docker-registries/index.md) in Octopus Deploy for a Docker registry. 
-- Add a project and define a deployment process. 
-- Select a worker pool. 
-- Select to run on a container for worker. 
-- Choose the previously added container registry. 
-- Enter an image name if already known. (e.g. !docker-image <octopusdeploy/worker-tools:ubuntu.18.04>)
-- Save.
-- Create release & deploy.
+## How to use execution containers for workers 
+
+- Set a [feed](/docs/packaging-applications/package-repositories/docker-registries/index.md) in Octopus Deploy for a Docker registry
+  - [Add Docker Hub as an external feed](https://octopus.com/blog/build-a-real-world-docker-cicd-pipeline#add-docker-hub-as-an-external-feed)
+- Add a project and define a deployment process
+- Set the `Execution Location` for your step to `Run on a worker`
+- In `Container Image` select `Runs on a worker inside a container`
+- Choose the previously added container registry
+- Enter an image name if already known. (e.g. !docker-image <octopusdeploy/worker-tools:1.0.1-ubuntu.18.04>)
```

Comment:
Docsync seems to be having trouble with this: !docker-image <octopusdeploy/worker-tools:1.0.1-ubuntu.18.04> 
Is there something pending for this include to work?

Reply:
Yes :) @emilol is fixing that soon.